### DISCUSSION TITLE
VisualShader: Add options to frame multiple node groups from right click menu

### DISCUSSION THREAD

**Discussion by Foyezes** [godotengine/godot-proposals#11204] [GUI]
Adding an option to frame multiple selected nodes from the right click menu, instead of having to create a new frame and dragging the nodes in.
![image](https://github.com/user-attachments/assets/0d9897a0-b9bc-42f9-9dd2-f582bfd666de)

**Comment by tetrapod00** [comment]
There is already a full-fledged proposal for this: https://github.com/godotengine/godot-proposals/issues/9868. I'm going to close this discussion and consolidate into that proposal. Thanks for taking the time to submit this!
